api/envoy/extensions/filters/http/geoip/v3/geoip.proto

```diff
@@ -82,9 +36,10 @@ message Geoip {
   // [#next-free-field: 2]
   XffConfig xff_config = 1;
 
-  // Configuration for geolocation headers to add to request.
-  GeolocationHeadersToAdd geo_headers_to_add = 2 [(validate.rules).message = {required: true}];
```

Comment:
Note to future reviewers: Although this is a breaking API change, it is allowed. This is because the filter is still WIP.